### PR TITLE
Update CHANGELOG for release 1.7.0

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: services/horizon/CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,19 @@
...
-## Unreleased
-
+## v1.7.0
+
+* Update Go toolchain to 1.14.6 in order to fix [golang/go#34775](https://github.com/golang/go/issues/34775),
+  which caused some database queries to be executed instead of rolled back.
+ 
+* Fix panic on missing command line arguments [#2872](https://github.com/stellar/go/pull/2872)
+
+* DB schema changes:
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md:13]: Can we make it a separate section on top (so it's not easy to miss) and include time duration of a migration? In staging server with full history it took around 5 minutes. See v1.1.0 for an example.

File: services/horizon/CHANGELOG.md [hunk 2]
```diff
@@ -3,8 +3,19 @@
 All notable changes to this project will be documented in this
 file. This project adheres to [Semantic Versioning](http://semver.org/).x
 
-## Unreleased
-
+## v1.7.0
+
+* Update Go toolchain to 1.14.6 in order to fix [golang/go#34775](https://github.com/golang/go/issues/34775),
+  which caused some database queries to be executed instead of rolled back.
+
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md:10]: Any reason for empty lines? It looks different when rendered in HTML compared to other versions.

Screenshot 2020-08-04 at 17 26 09

Comment (2opremio) [reply services/horizon/CHANGELOG.md:10]: I just blindly added the extra space. Will fix

**Comment by bartekn** [comment]
@2opremio or @tamirms can you include this in CHANGELOG: https://github.com/stellar/go/pull/2877?

**Pull request merged**